src/main/webapp/ListsOfUsers.jsp

```diff
@@ -16,18 +24,30 @@ table {
 
 th, td {
 	border: 1px solid black;
-	padding: 8px;
+	padding: 8rem;
```

Comment:
Please check the UI part. Because of 1 rem = 16 pixels. So, here it should be 0.5rem instead of 8rem. Please change it everywhere.